lib/vector/source/vector.c

```diff
@@ -16,7 +16,6 @@
 static int clear(vector_t *this)
 {
     this->size = 0;
-    this->available_size = this->total_size;
```

Comment:
Why ?

Reply:
available_size doesn't exist anymore on main branch, it's a fix